staging/src/k8s.io/pod-security-admission/cmd/webhook/server/server.go

```diff
@@ -171,7 +171,7 @@ func (s *Server) HandleValidate(w http.ResponseWriter, r *http.Request) {
 		defer cancel()
 	}
 
-	if r.Body == nil {
+	if r.Body == http.NoBody {
```

Comment:
just in case there are callers invoking this directly from non-server paths:

```suggestion
	if r.Body == nil || r.Body == http.NoBody {
```